maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -293,22 +284,21 @@ class IOSDriver(
     }
 
     override fun scrollVertical() {
-        val screenWidth = widthPixels ?: throw IllegalStateException("Screen width not available")
-        val screenHeight = heightPixels ?: throw IllegalStateException("Screen height not available")
+        val appId = activeAppId() ?: return
 
         iosDevice.scroll(
-            xStart = screenWidth / 4,
-            yStart = screenHeight / 4,
-            xEnd = screenWidth / 4,
-            yEnd = 0,
-            durationMs = IdbIOSDevice.DEFAULT_SWIPE_DURATION_MILLIS,
-            scrollType = IdbIOSDevice.ScrollType.SCROLL
+            appId = appId,
+            xStart = 0.5f,
+            yStart = 0.5f,
+            xEnd = 0.5f,
+            yEnd = 0.25f,
+            velocity = null
```

Comment:
Let's take this separately but let's keep this delta and duration configurable for scrolling as well.